download.html

```diff
@@ -32,11 +32,11 @@ <h3><a
         <div class="codebox">
           97CF851C947F1C5F3DC5E78E3756B55CBB27E1DC9EA658C36A4F536B37510A50
         </div>
-        <h3> <a href="https://github.com/mimblewimble/grin/releases/download/v5.1.0/grin-v5.1.0-win-x64.zip">grin
+        <h3> <a href="https://github.com/mimblewimble/grin/releases/download/v5.1.2/grin-v5.1.2-win-x64.zip">grin
             v5.1.0</a></h3>
```

Comment:
the versions here also need to be updated. Check https://github.com/mimblewimble/site/pull/251/files for example of a previous version update

Reply:
Corrected to 5.1.2 version for all oses.